Explorer/Assets/Scripts/CrdtEcsBridge/Engine/EngineAPIImplementation.cs

```diff
@@ -252,6 +258,20 @@ public void SetIsDisposing()
             isDisposing = true;
         }
 
+        public async UniTask<SendEthereumMessageResponse> SendEthereumMessageAsync(string method, object[] @params, CancellationToken ct)
```

Comment:
Every API domain should be in its own file(-s). `EngineAPI` has nothing to do with `EtheriumAPI`